### DISCUSSION TITLE
unable to install dependencies through npm install

### DISCUSSION THREAD

**Discussion by lakshmimaddipati** [js-fns/date-fns#3785] [Q&A] [answered]
Hi, I am trying to install the dependences and get this running on my system, but this error won't go away. Does this mean I need a linux machine to work on this? Any help is much appreciated. Was stuck on this for hours.
![image](https://github.com/date-fns/date-fns/assets/82107355/daa9fc44-b7d9-423d-91c3-ed39c20e0591)

**Comment by fturmel** [comment, marked as answer]
You mean the dev dependencies to contribute, right?

I think the problem is that the lock file right now is on Bun 1.0.30, but they only shipped Windows compat recently with v1.1 https://bun.sh/blog/bun-v1.1#windows-support. The pre-built binaries in 1.0.x were only for macOS (darwin) and linux.

Try running `npm update` instead of `npm install`, or delete the package lock file before `npm install`.

**Reply by lakshmimaddipati** [reply]
Yes, deleting the package-lock worked. I think this version incompatibility should be made clear in the documentation. Thanks for your reply :)

**Reply by fturmel** [reply]
The Bun dependency was added somewhat recently, and is only really needed for some extensive CI tests. We can resolve this issue by simply updating this specific dependency in the lock file now that Bun is shipping a win32 binary as well.

Fix incoming in #3787
